README.md

```diff
@@ -25,14 +25,50 @@ The service is composed by the following components:
 - *fathom-proxy*: the [nginx](https://www.nginx.com) service used as reverse-proxy
 
 ## Deployment / Configuration
-The templates provided in the `openshift` folder include everything that is necessary to create the required builds and deployments.
+The templates provided in the `openshift` folder include everything that is necessary to create the required builds and deployments.  
 
-Since there are interdependencies between deployment configurations, please make sure to follow this order when creating them for the first time:
-1) build and deploy the database
-2) build and deploy the Fathom analytics server
-3) build and deploy the proxy
+To run Fathom on openshift you **MUST** install [openshift-developer-tools](https://github.com/BCDevOps/openshift-developer-tools) and have them available on your path  
+
+By default, Fathom uses the artifactory docker registry. If you are going to keep the default settings artifactory **MUST** be enabled in your OCP cluster, otherwise you will have to tweak the param file to specify your docker registry.  
+
+
+#####Running with Artifactory:
+There should already be a "artifacts-default-\*\*\*\*\*\*" secret in the tools environment of your openshift cluster. Copy the username and password of this 
+secret and run the following command in each environment you wish to build/deploy to. (example: tools and dev / tools and prod)
+~~~
+oc create secret docker-registry artifactory-creds \
+    --docker-server=docker-remote.artifacts.developer.gov.bc.ca \
+    --docker-username=<our username from secret> \
+    --docker-password=<our password from secret> \
+    --docker-email=unused
+oc secrets link default artifactory-creds --for=pull
+oc secrets link builder artifactory-creds
+~~~
+
+#####Running with custom Docker registry:
+Pretty much the same command works when using a custom docker registry. You can change docker.io to be any server you want. If you're using the default unauthenticated docker registry, you can skip this section as long as you remeber to set DOCKER_REG and PULL_CREDS to blank and uncomment them.
+~~~
+oc create secret docker-registry docker-creds \
+    --docker-server=docker.io \
+    --docker-username=<docker username> \
+    --docker-password=<docker password> \
+    --docker-email=unused
+oc secrets link default docker-creds --for=pull
+oc secrets link builder docker-creds
+~~~
```

Comment:
What do you think about linking the docs that can be found [here](https://developer.gov.bc.ca/Artifact-Repositories) rather than having the full procedure in our readme?

Reply:
true, but I wanted to explain the artifactory thing in more detail. The origional github issue could have been a bit more detailed. EX: it doesn't explain that you already have an artifactory cred that needs to be copied

Reply:
also the github doc tells users to edit the json configuration, where as we just want users to edit the local.param files

Reply:
Yeah, the link I posted is to the official docs which have been cleaned up a bit. In theory users should not need to edit the configs at all - I agree with that, and not having the docs in here seems like a better idea (to me) so that we will not need to maintain them if they change again - we just reference the official docs.

What does @WadeBarnes think?

Reply:
The less documentation needed to be maintained in separate locations the better.